test/telephone_number/parser_test.rb

```diff
@@ -2,113 +2,37 @@
 
 module TelephoneNumber
   class ParserTest < Minitest::Test
-    class Consumer; include TelephoneNumber::Parser; end;
-
     def setup
-      @consumer = Consumer.new
+      @valid_numbers = YAML.load_file('test/valid_numbers.yml')
+      @invalid_numbers = YAML.load_file('test/invalid_numbers.yml')
     end
 
     def test_sanitize_removes_all_non_numeric_characters
       numbers = %w(555$$@@5555555 ##$#%#5555555555)
```

Comment:
I suppose we could do this explicitely, but keep in mind it's already tested with dashes in all of the cases in the `valid_numbers.yml`